accounts/views.py

```diff
@@ -40,3 +50,109 @@ class UserLoginView(LoginView):
 
 class UserLogoutView(LogoutView):
     pass
+
+
+class UserProfileView(LoginRequiredMixin, DetailView):
+    template_name = "accounts/profile.html"
+    model = Profile
+
+    def get_queryset(self):
+        return super().get_queryset().select_related("user")
```

Comment:
querysetが固定値であれば，これで良さそう
```suggestion
    queryset = Profile.objects.all().select_related("user")
```